File: src/test/java/com/tibiawiki/domain/utils/PropertiesUtilTest.java
package com.tibiawiki.domain.utils;

import org.junit.jupiter.api.Test;

import static org.hamcrest.MatcherAssert.assertThat;
import static org.hamcrest.Matchers.is;

public class PropertiesUtilTest {

    @Test
    void testGetUsername_Success() {
        String result = PropertiesUtil.getUsername();

        assertThat(result, is("Foo"));
    }

    @Test
    void testGetPassword_Success() {
        String result = PropertiesUtil.getPassword();

        assertThat(result, is("Bar"));
    }
}

File: src/test/kotlin/com/tibiawiki/domain/utils/PropertiesUtilTest.kt
package com.tibiawiki.domain.utils

import org.hamcrest.MatcherAssert.assertThat
import org.hamcrest.Matchers.`is`
import org.junit.jupiter.api.Test

class PropertiesUtilTest {

    @Test
    fun testGetUsername_Success() {
        assertThat(PropertiesUtil.getUsername(), `is`("Foo"))
    }

    @Test
    fun testGetPassword_Success() {
        assertThat(PropertiesUtil.getPassword(), `is`("Bar"))
    }
}
